src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -747,8 +747,8 @@ public void Roundtrip_CrossProcess(object obj)
             }
         }
 
-        [ActiveIssue(16753)] //Fails on desktop and core: 'Unable to cast object of type 'System.UInt32[][*]' to type 'System.Object[]'
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
@weshaggard, as noted in the comment on the ActiveIssue, the existing core behavior does match the existing desktop behavior: it fails with an exception "'Unable to cast object of type 'System.UInt32[][*]' to type 'System.Object[]'"